sympy/codegen/tests/test_cnodes.py

```diff
@@ -26,14 +27,19 @@ def test_goto_Label():
     g = goto(s)
     assert g.func(*g.args) == g
     assert g != goto('foobar')
-    assert ccode(g) == 'goto early_exit'
-
-    l = Label(s)
-    assert l.is_Atom
-    assert ccode(l) == 'early_exit:'
-    assert g.label == l
-    assert l == Label(s)
-    assert l != Label('foobar')
+    assert ccode(g) == 'goto early_exit;'
+
+    l1 = Label(s)
+    assert ccode(l1) == 'early_exit:'
+    assert l1 == Label(s)
+    assert l1 != Label('foobar')
+
+    body = [PreIncrement(x)]
+    l2 = Label(s, body)
+    assert l2.name == String("early_exit")
+    assert l2.body == CodeBlock(*body)
```

Comment:
I think it'll be better to test the complete expression

Also, add a test for multiple statements in the label body.

Reply:
Alright! But multiple statements are generally not used with label statements, right? Nevertheless, we can use something like this if we want to use multiple statements:
<pre>
label: 
{
    a++;
    b--;
}
</pre>

What do you think?

Reply:
Yeah, looks good. 

But you still need to test this with the complete body expression instead of * body.